ci-tools/github-runner/deployments/README.md

```diff
@@ -0,0 +1,140 @@
+# Setup GitHub App
+
+* Visit `https://github.com/organizations/ORG_NAME_HERE/settings/apps`,
+  replacing ORG_NAME_HERE with your organization's name.
+
+* Click "New GitHub app", and give the app a name, perhaps something like "GHA
+  runners on GCP". To help people understand what this is, set the "Homepage
+  URL" to https://github.com/chipsalliance/caliptra-sw/tree/main/ci-tools/github-runner
+
+* Uncheck the Webhook "Active" check-box. Once the project is fully deployed we
+  will fill in the callback URL later.
+  
+* Generate a random string and put it in the "Webhook secret" section. We will
+  need to store this string later in the GCP secret manager.
+
+* In "Repository Permissions, select the following:
+
+  * Actions - "Read-only"
+  * Metadata - "Read-only"
+  * Set everything else to "No access"
+
+* In Organization Permissions
+
+  * Self-hosted runners: "Read and Write"
+
+* Under "Subscribe to events", check the following
+
+  * Meta
+  * Workflow job
+
+* Under "Where can this GitHub App be installed", select "Only on this account"
+
+* Press "Create GibHub App"
+
+* Scroll down to "Private keys", and press "Generate a private key", and keep
+  the downloaded file to be uploaded into the GCP Secret Manager later.
+
+* Click on "App managers". Add any users you trust to manage this app for you.
+
+* Click on "Install App"
+
+* Select "Only select repositories"
+
+* Choose the repos that want to use these test runners.
+
+* Click Install
+
+Prod app is https://github.com/organizations/chipsalliance/settings/apps/caliptra-gha-runners-on-gcp
+ 
+# Setup application-default permissions
+
+This creates temporary credentials terraform can use to access your account:
+
+```
+gcloud auth application-default login
+```
+
+# Initial project setup
+
+First, create a file similar to deployments/env/kor.tf for your
+environment-specific variables. Copy the GitHub App ID from your GitHub App.
+
+Next, we need to setup the bucket used for the main tfstate.
+
+```
+cd deployments/init
+terraform init -var-file=../env/kor.tfvars 
+terraform apply -var-file=../env/kor.tfvars -state=kor.tfstate
```

Comment:
Could you help me understand why these files use your name?

Reply:
Laziness. Replaced with ${ENV_NAME}.tfvars.